o-fish-ios/Helpers/NotificationManager.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  NotificationManager.swift
+//  
+//  Created on 7/8/20.
+//  Copyright © 2020 WildAid. All rights reserved.
+//
+
+import UserNotifications
+
+class NotificationManager {
+
+    static let shared = NotificationManager()
+
+    private let notificationCenter = UNUserNotificationCenter.current()
+
+    func requestAutorization() {
+        notificationCenter.requestAuthorization(options: [.alert, .sound, .badge]) { (didAllow, error) in
+            if !didAllow {
+                print("User has declined notifications")
+            }
+
+            if let error = error {
+                print(error.localizedDescription)
+            }
```

Comment:
should we add extra code to handle notification send when user declined ?

Reply:
We can add alert with propose for user turns on notification, but I don't know, where and when should be this alert.

Reply:
Regarding Apple policy we prohibited to use such alerts

Reply:
I can add such alerts after changing on/off duty state (from off to on) with proposal turn on notification and link to settings. How many times should I use this alert?